test/utest/src/cuter_cerl_tests.erl

```diff
@@ -6,60 +6,43 @@
 -include("include/eunit_config.hrl").
 -include("include/cuter_macros.hrl").
 
--spec test() -> 'ok' | {'error', term()}.  %% This should be provided by EUnit
+-spec test() -> any().
```

Comment:
Why lie (overapproximate) in the return type?

Reply:
Done.